docs/source/installation/2-manual-install.rst

```diff
@@ -577,7 +577,9 @@ Run on your **local machine** to check if you have an SSH key added to your ssh
 
 If you don't see SSH key listed here for the user you are using to log in, you will need to add it to the ssh agent before running the ssh command.
 You can read about it here, https://www.ssh.com/academy/ssh/agent#adding-ssh-keys-to-the-agent.
-**WARNING**: ssh-add should only be run locally and not on the remote machine.
+
+.. warning::
+ssh-add should only be run locally and not on the remote machine.
```

Comment:
Oh, it's just missing an indent and a newline. e.g.

```restructuredtext
.. warning::
   ssh-add should only be run locally and not on the remote machine.

```

([Docs](https://docutils.sourceforge.io/docs/ref/rst/directives.html#warning))